.properties

```diff
@@ -1,3 +1,3 @@
 {
 	#format : #filetree
-}
```

Comment:
Hmm, do I need to remove a newline from the file contents that gets written by Squot?